tests/driver_my9221/main.c

```diff
@@ -45,16 +45,16 @@ int main(void)
     }
     /* run some tests */
     LOG_INFO("- light up all LEDs one by one.\n");
-    for (unsigned i=0; i < dev.params.leds; ++i) {
+    for (unsigned i=0; i < dev.params.leds; i++) {
         my9221_set_led(&dev, i, MY9221_LED_ON);
         xtimer_usleep(TEST_WAIT);
         my9221_set_led(&dev, i, MY9221_LED_OFF);
     }
     xtimer_usleep(TEST_WAIT);
-    for (unsigned i=dev.params.leds; i > 0 ; --i) {
-        my9221_set_led(&dev, i, MY9221_LED_ON);
+    for (unsigned i=dev.params.leds; i > 0 ; i--) {
```

Comment:
also unrelated

Reply:
I would rather change the boundaries, because in the same function `i` was first a led number, then a led +1.
But as it is an unsigned so of `>= 0` cannot be used.

What is our normal way of handling these ?
Iterate on `j` and define `i = j - 1` as an intermediate variable ?

Reply:
that's why I prefer the `while(i--)` pattern for reverse counts :)

But yeah, it's cleaner with an intermediate variable. I will push it.